custom/inddex/reports/gaps_detail.py

```diff
@@ -44,16 +46,27 @@ def __init__(self, food_data):
 
     @property
     def rows(self):
-        return []
+        rows = {}
+        for row in self._food_data.rows:
+            if row.conv_factor_gap_code != ConvFactorGaps.AVAILABLE:
+                key = (row.food_name, row.conv_method_code, row.conv_factor_gap_code)
```

Comment:
IMO this doesn't make a lot of sense, as it only considers conversion factor gaps, but that's what was specified, so we'll start like this.
:shrug: